game/ui/city/scorescreen.cpp

```diff
@@ -156,11 +164,19 @@ void ScoreScreen::setFinanceMode()
 		    ->setText(format("$%d", agentsSalary + basesCosts));
 
 		int balance = state->getPlayer()->balance;
+		
+		// Special case: when shown during weekly upkeep balance was already adjusted by the game loop
```

Comment:
The clang-format version in the CI wants this comment line to be broken, only a small format change but keeping that happy makes it easier to spot more important issues :)